scripts/sysman/rasState.yml

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Ras Get State"
+version: "1.7"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "RAS Get State Extension Name"
+version: "1.7"
+name: $S_RAS_GET_STATE_EXT_NAME
+value: '"$XS_extension_ras_state"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "RAS Get State Extension Version(s)"
+version: "1.7"
+name: $s_ras_state_ext_version_t
+etors:
+    - name: "1_7"
+      value: "$X_MAKE_VERSION( 1, 7 )"
+      desc: "version 1.7"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension structure for providing RAS error counters for different error sets"
+version: "1.7"
+class: $sRas
+name: $s_ras_state_ext_t
+members:
+    - type: $s_ras_error_cat_t
+      name: "category"
+      desc: "[out] category for which error counter is provided."
+    - type: uint64_t
+      name: errorCounter
+      desc: "[out] Current value of RAS counter for specific error category."
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Ras Get State"
+class: $sRas
+name: RasGetStateExt
+details:
+    - "This function retrieves error counters for different RAS error categories."
+    - "Clearing errors will affect other threads/applications - the counter values will start from zero."
```

Comment:
who clears the errors?

Reply:
ah, we have the clear below. So this function does two things: get errors, but also clear. Do we want to have two tasks in a single API, or better to have a different API to clear? Main issue with having both in the same function is that now it cannot be actually called from simultaneous threads, unless there's a lock inside the L0 driver.

by having two, we can indeed having multiple threads calling this getter, as all they would be doing is to retrieve informantion, w/o changing state.

Moreover, clear requires privileges?

Reply:
sure, agree it's cleaner. the existing API for `zesRasGetState` does bundle it together and I just followed that here. but yes, I can separate this out.

Reply:
@jandres742 added separate ext to clear errors now.